atcoder/modint.hpp

```diff
@@ -91,6 +91,7 @@ struct static_modint : internal::static_modint_base {
 
     mint operator+() const { return *this; }
     mint operator-() const { return mint() - *this; }
+    operator bool() const { return _v != 0; }
```

Comment:
```suggestion
    explicit operator bool() const { return _v != 0; }
```
avoid implicit conversion